tests/python_api/numpy_reader.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import random
+
+from amd.rocal.pipeline import Pipeline
+from amd.rocal.plugin.generic import ROCALNumpyIterator
+import amd.rocal.fn as fn
+import sys
+import os
+
+def draw_patches(image, idx, layout="nhwc", dtype="uint8"):
+    # image is expected as a numpy array
+    import cv2
+    if layout == "nchw":
+        image = image.transpose([1, 2, 0])
+    if dtype == "fp16":
+        image = image.astype("uint8")
+    image = cv2.cvtColor(image, cv2.COLOR_RGB2BGR)
+    cv2.imwrite("output_folder/numpy_reader/" + str(idx) + ".png", image)
+
+def main():
+    if len(sys.argv) < 3:
+        print('Please pass numpy_folder cpu/gpu batch_size')
+        exit(0)
+    data_path = sys.argv[1]
+    try:
+        path = "output_folder/numpy_reader/"
+        isExist = os.path.exists(path)
+        if not isExist:
+            os.makedirs(path)
+    except OSError as error:
+        print(error)
+    if (sys.argv[2] == "cpu"):
+        rocal_cpu = True
+    else:
+        rocal_cpu = False
```

Comment:
```suggestion
    if (sys.argv[2] == "cpu"):
        rocal_cpu = True
    else:
        rocal_cpu = False
```
rocal_cpu = True if sys.argv[2] == "cpu" else False